src/CommitmentsV2/SFA.DAS.CommitmentsV2/SFA.DAS.CommitmentsV2.csproj

```diff
@@ -24,6 +24,7 @@
     <PackageReference Include="SFA.DAS.Apprenticeships.Api.Client" Version="0.11.233" />
     <PackageReference Include="SFA.DAS.Apprenticeships.Api.Types" Version="0.11.233" />
     <PackageReference Include="SFA.DAS.Authorization.Features" Version="4.3.1" />
+    <PackageReference Include="SFA.DAS.CommitmentsV2.Shared" Version="4.1.664" />
```

Comment:
This should be referencing the project directly and not the shared package (sorry missed this earlier)